### ISSUE TITLE
Correct supportedKindsOfLldpAgentList from leaf to leaf-list in YANG

### ISSUE THREAD

**Issue by openBackhaul** [openBackhaul/lldp#12]
supported-kinds-of-lldp-agent-list should be leaf-list instead of leaf in YANG.
For unknown reason supportedKindsOfLldpAgentList was already [*] in UML.
Is there a UML2YANG translation problem?

**Comment by PrathibaJee** [comment]
Yes, this is an issue in the UML2YANG tool. Fixed the same in the tool. 
Please find the details in the corresponding [issue#52](https://github.com/Open-Network-Models-and-Interfaces-ONMI/onmi-iisomi-xhaul-uml-yang/issues/52) in the UML2YANG tool repository.

**Comment by openBackhaul** [comment]
Bug in existing translation of IETF datatypes has been fixed.
YANG was already fixed.
Many thanks to Prathiba.
